mmdet/datasets/pipelines/formating.py

```diff
@@ -213,7 +213,7 @@ def __call__(self, results):
                 continue
             results[key] = DC(to_tensor(results[key]))
         if 'gt_masks' in results:
-            results['gt_masks'] = DC(results['gt_masks'], cpu_only=True)
+            results['gt_masks'] = DC(to_tensor(results['gt_masks']), cpu_only=True)
```

Comment:
Please do not submit a commit that is not part of this PR.

Reply:
Can not be modified.